internal/convert/registryv1.go

```diff
@@ -158,17 +158,13 @@ func validateTargetNamespaces(supportedInstallModes sets.Set[string], installNam
 			return nil
 		}
 	default:
-		if supportedInstallModes.Has(string(v1alpha1.InstallModeTypeMultiNamespace)) {
+		if supportedInstallModes.Has(string(v1alpha1.InstallModeTypeMultiNamespace)) && !set.Has("") {
```

Comment:
If we aren't going to support the `""` value at all, we should fail fast with validation logic when attempting to create the resource.

Reply:
We do support targetNs with a single empty string element to handle watching all Ns. The CEL validation which has been added should along with this check should take care of all the cases.